src/test/java/uk/gov/companieshouse/fixtures/DissolutionFixtures.java

```diff
@@ -227,4 +229,25 @@ public static DissolutionUserData generateDissolutionUserData() {
 
         return userData;
     }
+
+    public static RefundResponse generateRefundResponse() {
+        final RefundResponse refundResponse = new RefundResponse();
+
+        refundResponse.setAmount(800);
+        refundResponse.setCreatedDateTime(LocalDateTime.now().toString());
+        refundResponse.setRefundId("GTY890");
+        refundResponse.setStatus("success");
+
+        return refundResponse;
+    }
+
+    public static RefundInformation generateRefundInformation() {
+        final RefundInformation refundInformation = new RefundInformation();
+
+        refundInformation.setAmount(800);
+        refundInformation.setCreatedDateTime(LocalDateTime.now().toString());
+        refundInformation.setRefundId("GTY890");
+
+        return refundInformation;
+    }
```

Comment:
generateRefundResponse should be in PaymentFixtures. Wondering if generateRefundInformation should stay here or also go into PaymentFixtures? 🤔